tests/jmeter/jmeter-fake.py

```diff
@@ -27,24 +27,18 @@ def udp_server():
 
 
 def files():
-    import shutil
     artifacts_dir = get_artifacts_dir()
     jmeter_path = os.path.dirname(__file__)
     jtl_file = jmeter_path + '/jtl/tranctl.jtl'
-    file_name = 'kpi'
-    suffix = ''
-    i = 0
-    while os.path.isfile(os.path.join(artifacts_dir, file_name + suffix + '.jtl')):
-        i += 1
-        suffix = '-%i' % i
-    shutil.copy(jtl_file, os.path.join(artifacts_dir, file_name + suffix + '.jtl'))
+    jmx = [x for x in sys.argv if x.endswith('.jmx')][0]
```

Comment:
Is it really worth it to create a complete JMeter-compatible option parser (because optparse will raise if it sees arguments that are unknown to him) just to grab a one JMX argument?